pkgs/games/gl-gsync-demo/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, libX11, freeglut, glew, libXNVCtrl, libXext, lib }:
+
+let
+  pname = "gl-gsync-demo";
+
+  version = "2019-05-20";
+
+  src = fetchFromGitHub {
+    owner = "dahenry";
+    repo = "gl-gsync-demo";
+    rev = "0d45648d1af131cb7691008a0165f9464793f95f";
+    sha256 = "0mj728b0a3sh4624rp8wbzd8s838bvdp5l9ab72p16pkcn3my7z6";
+  };
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "gl-gsync-demo";
  version = "2019-05-20";

  src = fetchFromGitHub {
    owner = "dahenry";
    repo = "gl-gsync-demo";
    rev = "0d45648d1af131cb7691008a0165f9464793f95f";
    sha256 = "0mj728b0a3sh4624rp8wbzd8s838bvdp5l9ab72p16pkcn3my7z6";
  };
```

Reply:
I prefer the let in inherit pattern over rec.

Reply:
and I think this let in just creates noise here.